ignite/contrib/handlers.py

```diff
@@ -0,0 +1,68 @@
+try:
+    from tqdm import tqdm
+except ImportError:
+    raise RuntimeError("This contrib module requires tqdm to be installed")
+
+from ignite.engine import Events
+
+
+class ProgressBar:
+    """
+    TQDM progress bar handler to log training progress and computed metrics
+
+    Args:
+        engine: ignite.Engine object
+        loader: iterable or dataloader object
+        output_transform: transform a function that transforms engine.state.output
+                into a dictionary of format {name: value}
+
+    Example:
+        (...)
+        pbar = ProgressBar(trainer, train_loader, output_transform=lambda x: {'loss': x})
+        trainer.add_handler(Events.ITERATION_COMPLETED, pbar)
+    """
+
+    def __init__(self, engine, loader, output_transform=lambda x: x):
+        self.num_iterations = len(loader)
```

Comment:
nit: maybe we can setup `num_iterations` directly instead of `loader` ?

Reply:
sure, but I don't see the advantage of that... the user will have the loader more easily accessible, and from an efficiency point of view, it is the same

Reply:
For me if handler require data loader than it will work with its content. So, without looking into the code user can have a misleading a priori on the handler

Reply:
sure, I don't mind going with num_iterations instead but bear in mind that from the user point of view this is more confusing because it's not the total number of iterations, but rather the number of batches in the loader